src/Symfony/Component/RemoteEvent/Messenger/ConsumeRemoteEventHandler.php

```diff
@@ -33,6 +33,12 @@ public function __invoke(ConsumeRemoteEventMessage $message): void
         }
         $consumer = $this->consumers->get($message->getType());
 
-        $consumer->consume($message->getEvent());
```

Comment:
that's strange, why don't we enforce consumer implement ConsumerInterface instead?

Reply:
I don't know! cc @fabpot

Reply:
See #50093